src/main/webapp/app/exam/manage/exams/exam-checklist-component/exam-checklist.component.html

```diff
@@ -115,92 +122,117 @@ <h3>
                     </a>
                 </td>
             </tr>
+            <ng-container *ngIf="!isTestExam">
+                <tr>
+                    <td>3</td>
+                    <td>
+                        <span>{{ 'artemisApp.examManagement.checklist.tableItem.register' | artemisTranslate }}</span>
+                    </td>
+                    <td>
+                        <span>{{ 'artemisApp.examManagement.checklist.descriptionItem.register' | artemisTranslate }}</span>
+                        <hr />
+                        <ul class="list-unstyled">
+                            <li class="list-group-item border-1">
+                                <span>
+                                    <jhi-exam-checklist-check [checkAttribute]="!!exam.numberOfRegisteredUsers && exam.numberOfRegisteredUsers! > 0"></jhi-exam-checklist-check>
```

Comment:
```suggestion
                                    <jhi-exam-checklist-check [checkAttribute]="!!exam.numberOfRegisteredUsers"></jhi-exam-checklist-check>
```
This should be sufficient, or?

Reply:
I think this was just taken over from the existing code. The changes here do not recognize that the code was there before and was just wrapped in an additional ngIf statement. I would just keep it and if at all change it in a follow up